portfolio/src/main/webapp/script.js

```diff
@@ -80,3 +80,9 @@ function hasWon(){
     winBox.innerText = "";
     }
 }
+
+async function getHello() {
+  const response = await fetch('/data');
+  const hello = await response.text();
+  document.getElementById('hello-container').innerText = hello;
+}
```

Comment:
Tiny nit: if you add a new line at the very end of this file (just after the `}`) that little red icon will go away